api-testing/multicloud-api/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
+group 'sodafoundation.io'
+version '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
Are all these dependencies taken for the latest releases artifacts?

Reply:
Yes.